perms/perms.go

```diff
@@ -185,5 +185,6 @@ var (
 		"/universerpc.Universe/AssetLeaves":     {},
 		"/universerpc.Universe/QueryProof":      {},
 		"/universerpc.Universe/InsertProof":     {},
+		"/universerpc.Universe/Info":            {},
```

Comment:
Hmm, do we actually want people to be able to hit this?

Reply:
If we want this simple check to work, then yes, we need users to be able to connect directly without a macaroon and query this info. That's why I chose not to include any sensitive information. Maybe we even need to remove the two counts we have as those issue DB queries. If that is a concern and we just want to use this for the ID, maybe we could rename the RPC to just `ServerID`?

Reply:
> Maybe we even need to remove the two counts we have as those issue DB queries. 

In theory users can already obtain this information if the universe server is open, albeit with even more DB queries.